scripts/ocr_migration_apply.py

```
@@ -0,0 +1,683 @@
+        label_consolidated_from=n.get("label_consolidated_from"),
+    )
+
+
+# --------------------------------------------------------------------------- #
+# OCR + entity construction (ingestion conventions)                            #
+# --------------------------------------------------------------------------- #
+
+
+def run_swift_ocr(binary: Path, image_path: Path) -> dict[str, Any] | None:
+    out = tempfile.mkdtemp()
+    subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)